Formula/i/ipsw.rb

```diff
@@ -1,8 +1,8 @@
 class Ipsw < Formula
   desc "Research tool for iOS & macOS devices"
   homepage "https://blacktop.github.io/ipsw"
-  url "https://github.com/blacktop/ipsw/archive/refs/tags/v3.1.527.tar.gz"
-  sha256 "de451c36286d0fdeb21b0dd048f0bb6d362667f4ad7f8bc05ce25e34ee49d4dc"
+  url "https://github.com/blacktop/ipsw/archive/refs/tags/v20.08.91.tar.gz"
```

Comment:
oh this is a bit off.

Reply:
Not sure why this is popping up all of a sudden, but adding a `github_latest` `livecheck` strategy should fix it.